services/MouthShut.com.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "MouthShut",
+  "name": "MouthShut.com",
```

Comment:
I feel that MouthShut.com is the name of the company and that MouthShut is the name of the service. While MouthShut.com is also used in quite a few occasions, it looks like it is slightly less used. When in doubt and used interchangeably, I'd be in favor of preferring the “name” rather than the “domain name”. If you agree, let's add that to #106.

![Capture d’écran 2020-09-02 à 18 51 45](https://user-images.githubusercontent.com/222463/92013234-c6b16280-ed4d-11ea-9e46-266990ddddcf.png)
![Capture d’écran 2020-09-02 à 18 51 47](https://user-images.githubusercontent.com/222463/92013238-c749f900-ed4d-11ea-9f47-f1b824c9074f.png)
![Capture d’écran 2020-09-02 à 18 51 56](https://user-images.githubusercontent.com/222463/92013241-c7e28f80-ed4d-11ea-874f-b95c58e9bf07.png)
![Capture d’écran 2020-09-02 à 18 52 12](https://user-images.githubusercontent.com/222463/92013244-c7e28f80-ed4d-11ea-87b1-bb368cf90695.png)
![Capture d’écran 2020-09-02 à 18 53 12](https://user-images.githubusercontent.com/222463/92013255-ce710700-ed4d-11ea-820e-681162bfbd94.png)

Reply:
@Ndpnt so does the merge mean we agree to prefer the service name over the domain name when they are used interchangeably? 🙂

Reply:
Yep, sorry I though I already replied. 😬